src/Autocomplete/assets/src/controller.ts

```diff
@@ -325,10 +325,39 @@ export default class extends Controller {
         return string.replace(/(<([^>]+)>)/gi, '');
     }
 
-    #mergeObjects(object1: any, object2: any): any {
-        return { ...object1, ...object2 };
+    #mergeConfigs(config1: any, config2: any): any {
+        return {
+            ...config1,
+            ...config2,
+            // Plugins from both configs should be merged together.
+            plugins: {
+                ...this.#normalizePluginsToHash(config1.plugins || {}),
+                ...this.#normalizePluginsToHash(config2.plugins || {}),
+            },
```

Comment:
Converting `['foo']` or `[{ name: 'foo', options: {/* ... */} }]` to `{ foo: {/* ... */} }` helps us to merge the plugins configurations